cmd/check.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2017 The OPA Authors.  All rights reserved.
+// Use of this source code is governed by an Apache2
+// license that can be found in the LICENSE file.
+
+package cmd
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/open-policy-agent/opa/ast"
+
+	"github.com/spf13/cobra"
+)
+
+var checkCommand = &cobra.Command{
+	Use:   "check",
+	Short: "Check that rego code can parse and compile",
+	Long: `Check that a rego source file can be parsed and compiled.
+
+If the 'check' command succeeds in parsing and compiling the module, no output
+is produced. If the parsing or compiling fails, 'check' will output the errors
+and exit with a non-zero exit code.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		os.Exit(checkModules(args))
+	},
+}
+
+func checkModules(args []string) int {
+	var errors []string
+	modules := map[string]*ast.Module{}
+	checkModule := func(filename string, info os.FileInfo, err error) error {
```

Comment:
I'd like to see this file loading logic made re-usable. We currently have a file loading utility inside the `runtime` package.